media/css/pebbles/components/_buttons-download.scss

```diff
@@ -107,8 +107,7 @@ ul.download-list {
 .download-button .os_win64-msi,
 .download-button .os_linux,
 .download-button .os_linux64,
-// Uncomment this to enable the win64 download buttons:
-// .win7up.x86.x64 .download-button .os_win,
+.win7up.x86.x64 .download-button .os_win,
```

Comment:
@amychurchwell this represents a pretty large and untested change that effects the whole of the website, so we may need to be a little careful here.

As far as I know, 64bit download buttons have never been enabled by default for Windows, because we understand that relying on user agent detection is not entirely accurate when it comes to system architecture (I think the product teams may have some data here). There may also be memory requirements to consider too. So instead, we've been relying on the stub installer delivering the correct binary, which it can do more reliably than we can here on the website.

As part of bedrock's technical roadmap, I'd like to see us try and reduce our reliance on user agent strings as much as is technically possible. We are seeing clear signals from other browsers that they intend to deprecate and freeze such information over time.

There is a suggestion in the original bug you linked to that may be worth exploring:

```
Maybe stub installer can pass along its detection result and make this happen?
```

If the architecture could be passed along via a query parameter to the stub installer page, then we may be able to work out a way to show the right build. Thoughts?

If/when we do enable 64bit builds by default for Windows, then we should probably do that as it's own distinct piece of work, and figure out the best way to implement and monitor it.

Reply:
Thank you for the info, Alex. Definitely do not want to enable by default.
I am going to separate these two issues because this feels larger scope & I would like to get a bugfix in for the dev edition & nightly CTA issue (#8579)